src/Core/Enums/ReferenceEventType.cs

```diff
@@ -28,5 +28,17 @@ public enum ReferenceEventType
         SendCreated,
         [EnumMember(Value = "send-accessed")]
         SendAccessed,
+        [EnumMember(Value = "organization-imported")]
+        DirectorySynced,
+        [EnumMember(Value = "vault-imported")]
+        VaultImported,
+        [EnumMember(Value = "first-secret-added")]
```

Comment:
Please remove "first-" from enum values/members and just leave them `SecretAdded`, `GroupCreated`, `CollectionCreated`